src/request.rs

```diff
@@ -283,6 +293,16 @@ impl<State> Request<State> {
         })
     }
 
+    /// Take the request body as a Body.
```

Comment:
I don't know if it is conventional over all of tides docs, but I feel like `Body` this should be in backticks (\`) as it references a rust type.